scss/_responsive.scss

```diff
@@ -0,0 +1,11 @@
+@media (max-width: $screen-mobile-max-width) {
```

Comment:
isn't it better to keep this "responsive" configurations inside elements? Like this:

```sass
body {
  // other attributes
  +@media (max-width: $screen-mobile-max-width) {
    padding: $em;
  }
}
```